database/migrations/2025_10_01_172830_create_appointments_table.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up(): void
+    {
+        Schema::create('appointments', function (Blueprint $table) {
+            $table->id();
+            $table->foreignId('clinic_id')->constrained('clinics', 'id')->cascadeOnDelete();
```

Comment:
I think this works without the text inside constrained or even something like ` $table->foreignIdFor(Clinic::class)->constrained()->cascadeOnDelete();`

Not really sure where light it standards lie in this matter

Reply:
It should be:

```suggestion
            $table->foreignId('clinic_id')
                ->constrained()
                ->cascadeOnDelete();
```

We prefer to abstract the migrations from the app's domain so we don't use the models here